indico/http/client.py

```diff
@@ -186,3 +194,165 @@ def _make_request(
                 error=error, code=response.status_code, extras=extras
             )
         return content
+
+
+class AIOHTTPClient(HTTPClient):
+    """
+    Beta client with a minimal set of features that can execute
+    requests using the aiohttp library
+    """
+
+    def __init__(self, config: Optional[IndicoConfig] = None):
+        """
+        Config options specific to aiohttp
+        unsafe - allows interacting with IP urls
+        """
+        self.config = config or IndicoConfig()
+        self.base_url = f"{self.config.protocol}://{self.config.host}"
+        unsafe = config.verify_ssl
+
+        self.request_session = aiohttp.ClientSession()
```

Comment:
thoughts on using httpx here instead of aiohttp? mainly because we're using httpx for flurry and insights

Reply:
maybe one concern is that httpx needs python 3.8+ and we require working in py 3.7?

Reply:
We dropped 3.7 since it's end of life https://devguide.python.org/versions/

Reply:
nice! well then I'll see if I can get httpsx working within a short period of time